File: Practice(Beginner)/C++/Mutated Minions.cpp
// Source: https://www.codechef.com/problems/CHN15A#

#include <bits/stdc++.h>
using namespace std;
int main(){
    int t, temp;
    cin >> t;
    while (t--){
        int n, v;
        cin >> n >> v;
        int c = 0;
        for (int i = 0; i < n; ++i) {
            cin >> temp;
            if((temp+v) % 7 == 0){
                c++;
            }
        }
        cout << c << endl;
    }
}

File: Practice(Beginner)/Python3/Mutated_Minions.py
# Source: https://www.codechef.com/problems/CHN15A#

t = int(input())

for test in range(t):
    n, v = map(int, input().split())
    l = list(map(int, input().split()))
    c = 0
    for i in l:
        if((i+v) % 7 == 0):
            c += 1
    print(c)